prow/knative/plugins.yaml

```diff
@@ -94,6 +94,13 @@ config_updater:
 blunderbuss:
   use_status_availability: true
 
+# Settings for the "trigger" plugin.
+triggers:
+- repos:
+  - knative:
```

Comment:
`:` will be evaluated as part of the string matching so it won't work, dropping it would do